app/models/checkout.rb

```diff
@@ -44,4 +51,41 @@ def card_number
   scope :old, -> { where('checked_in_at IS NOT NULL') }
   scope :current, -> { where('checked_in_at IS NULL') }
 
-end
+  private
+
+  def notify
+    if (self.checked_in_at != nil)
```

Comment:
You're calling this `after_update`, but is the only reason this could ever get updated a check-in? Otherwise you'll be sending this randomly. I think it would be much better to make this part of the controller. That way you could even add a "notify next" button on the frontend and have the controller action half-wired up.